src/Tribe/Tickets_Handler.php

```diff
@@ -753,26 +763,62 @@ public function save_image_header( $post_id ) {
 	/**
 	 * Save the current global stock properties for this event.
 	 *
+	 * @since TBD
+	 * Can come from ticket creation or settings edit. No longer tied to event save.
+	 *
 	 * @param int $post_id
 	 */
 	public function save_global_stock( $post_id ) {
-		if ( ! ( isset( $_POST[ 'tribe-tickets-post-settings' ] ) && wp_verify_nonce( $_POST[ 'tribe-tickets-post-settings' ], 'tribe-tickets-meta-box' ) ) ) {
-			return;
-		}
-
 		// Bail on autosaves/bulk updates
 		if ( wp_is_post_autosave( $post_id ) || wp_is_post_revision( $post_id ) ) {
 			return;
 		}
 
-		$enable = ! empty( $_POST[ 'tribe-tickets-enable-global-stock' ] );
-		$stock  = (int) @$_POST[ 'tribe-tickets-global-stock' ];
+		//if ( ! ( isset( $_POST[ 'tribe-tickets-post-settings' ] ) && wp_verify_nonce( $_POST[ 'tribe-tickets-post-settings' ], 'tribe-tickets-meta-box' ) ) ) {
```

Comment:
If we don't need the commented-out lines, could simply remove them? We can always look back in the version control history if we want to dig them up again.